apiservers/engine/backends/container.go

```diff
@@ -231,18 +249,50 @@ func (c *Container) ContainerStart(name string, hostConfig *container.HostConfig
 		// need to look at in hostConfig
 	}
 
-	// Start the container
+	// get a handle to the container
+	getRes, err := client.Containers.Get(containers.NewGetParams().WithID(name))
+	if err != nil {
+		if _, ok := err.(*containers.GetNotFound); ok {
+			return derr.NewRequestNotFoundError(err)
+		}
+
+		return derr.NewErrorWithStatusCode(err, http.StatusServiceUnavailable)
+	}
+
+	h := getRes.Payload
+
+	// bind network
+	bindRes, err := client.Scopes.BindContainer(scopes.NewBindContainerParams().WithHandle(h))
+	if err != nil {
+		if _, ok := err.(*scopes.BindContainerNotFound); ok {
+			return derr.NewRequestNotFoundError(err)
+		}
+
+		return derr.NewErrorWithStatusCode(err, http.StatusServiceUnavailable)
+	}
+
+	h = bindRes.Payload
+
+	// change the state of the container
 	// TODO: We need a resolved ID from the name
-	plStartParams := exec.NewContainerStartParams().WithID(name)
-	_, err := client.Exec.ContainerStart(plStartParams)
+	_, err = client.Containers.StateChange(containers.NewStateChangeParams().WithHandle(h).WithState("RUNNING"))
```

Comment:
If StateChange doesn't actually power on the VM directly then we should be using the handle that's returned from this call to pass into create.